refurbish/tests/Main.hs

```diff
@@ -58,8 +58,9 @@ data UseDockerRunner = UseDockerRunner Bool
 instance TO.IsOption UseDockerRunner where
   defaultValue = UseDockerRunner True
```

Comment:
hmm, the default was already to use the Docker runner. I don't object to inverting the name of the option, but we should probably change this if we do. I guess it is superseded by `optionCLParser`?